packages/vue-app/template/client.js

```diff
@@ -14,8 +14,13 @@ import {
   compile,
   getQueryDiff,
   globalHandleError
-} from './utils'
-import { createApp, NuxtError } from './index'
+} from './utils.js'
+import { createApp, NuxtError } from './index.js'
+import NuxtLink from './components/nuxt-link.client.js' // should be included after ./index.js
```

Comment:
Consistency: Use `.js` extension everywhere or nowhere for import

Reply:
In other files in vue-app it import with .js.

Reply:
If someday we want to directly serve templates in the browser (script with module type), `.js` is required. PS more explicit is always better and already has consistency benefit for us :)